python/tank/commands/clone_configuration.py

```diff
@@ -171,8 +171,8 @@ def _do_clone(log, tk, source_pc_id, user_id, new_name, target_linux, target_mac
     
     # copy files and folders across
     try:
-        os.mkdir(target_folder, 0777)
-        os.mkdir(os.path.join(target_folder, "cache"), 0777)
+        os.mkdir(target_folder, 0o777)
```

Comment:
didn't realise that this change was part of python3. So much clearer than the slightly magical 2.x syntax :)